samples-tests/src/test/kotlin/org/gradle/kotlin/dsl/samples/HelloKaptSampleTest.kt

```diff
@@ -0,0 +1,38 @@
+package org.gradle.kotlin.dsl.samples
+
+import org.hamcrest.CoreMatchers.allOf
+import org.hamcrest.CoreMatchers.containsString
+import org.junit.Assert.assertThat
+import org.junit.Test
+
+
+class HelloKaptSampleTest : AbstractSampleTest("hello-kapt") {
+
+    @Test
+    fun `hello kapt`() {
+        val output = build("run").output
+        println(output)
```

Comment:
❌ Could you remove this `println()` statement?

Reply:
Oh, sorry, forgotten debug